cmd/veneur-emit/main.go

```diff
@@ -241,9 +294,9 @@ func Main(args []string) int {
 		return 1
 	}
 	if span.TraceId != 0 {
-		if !flagStruct.ToSSF {
+		if !flagStruct.ToSSF && !flagStruct.ToGrpc {
 			logrus.WithField("ssf", flagStruct.ToSSF).
-				Error("Can't use tracing in non-ssf operation: Use -ssf to emit trace spans.")
+				Error("Can't use tracing in non-ssf operation: Use -ssf or -grpc to emit trace spans.")
```

Comment:
I'm not sure I understand this comment and the surrounding logic. Veneur doesn't have support for spans (tracing) that are not formatted using the SSF format. SSF is a hard requirement, and GRPC is not, no?

Reply:
Correct. I made this change early on when I didn't fully understand how this was working. You are right that it doesn't make sense so I will revert it.